dll/win32/userenv/lang/pt-PT.rc

```diff
@@ -17,9 +17,9 @@ BEGIN
     IDS_PROGRAMS "Menu Iniciar\\Programas"
     IDS_STARTUP "Menu Iniciar\\Programas\\Inicializar"
     IDS_LOCALSETTINGS "Configurações locais"
-    IDS_LOCALAPPDATA "Configurações locais\\Dados de aplicativos"
+    IDS_LOCALAPPDATA "Configurações locais\\Dados de Programas"
     IDS_TEMP "Configurações locais\\Temp" /* See also TEMP_DIR in boot/bootdata/hivedef.inf */
-    IDS_CACHE "Configurações locais\\Temporary Internet Files"
+    IDS_CACHE "Configurações locais\\Ficheiros temporários da Internet"
```

Comment:
This path is also localized in Windows XP/Windows 2003?

Reply:
On Windows XP yes. Windows2003 I don't know, but I suppose so.